innopoints/schemas/account.py

```diff
@@ -15,6 +15,11 @@ class Meta:
     balance = ma.Int()
 
 
+class TransactionSchema(ma.ModelSchema):
+    class Meta:
+        model = Transaction
```

Comment:
Don't omit the `ordered` and `sqla_session` fields. Maybe those are not required if all this schema is used in is Nested, but if we ever use it directly, we will need it

Reply:
I didn't omit them on purpose, I just didn't know what do they do 😅